### PR TITLE
Print total results banner at the end of execution

### LINKED ISSUE #155: Print total results banner at the end of execution
As a user, I would like to see the final result of my execution indicating whether the execution was successful or not. Additionally, it would be useful to see total execution time for all executed commands.

In case of failed execution – I would like to see the phase that failed and exit code just in case.

## Acceptance Criteria

1. introduce status banner at the end of the last executed phase – this banner appears always at the end – in case of successful execution or at the end of the failed phase
2. Example of a successful banner – width of the lines must be `72` characters:
```
------------------------------------------------------------------------
SUCCESS - Total time: 8s
------------------------------------------------------------------------
```
3. Example of a failed banner:
```
Execution failed: phase 'a' - exit code: 127
------------------------------------------------------------------------
FAILURE - Total time:  1m 53s
------------------------------------------------------------------------
```
> Notice that the line with `Execution failed` is a replacement of the current failure message. We are changing the format of the message to be more minimalistic.

4. `Success` and `Failure` keywords should be coloured as the following:
    - `SUCCESS` – `CYAN`
    - `FAILURE` – `RED`

5. Total time
    - In `min & sec`
    - example:
        - `9s`
        - `28s`
        - `1m 59s`
        - `7m` 

See the attached screenshot as an example.
1build_success
1build_failure

### PR DESCRIPTION
**Pull request by budevg**
## Description

Simple results banner which is printed at the end

## Fixes/Implements: # (issue)
This commit resolves #155 

## New dependencies introduced?
no

## Checklist:

- [ x ] I have added tests that prove my fix is effective or that my feature works
- [ x ] I have update README (If needed)

### CODE REVIEW

**Comment by gopinath-langote** [comment]
@budevg Please make sure your fork is up-to-date with the master.

**Comment by budevg** [comment]
ok I rebased the fork and force pushed the pr

**Comment by gopinath-langote** [comment]
Your current implementation prints following
```
------------------------------------------------------------------------
SUCCESS - Total Time: 6m28.367694s
------------------------------------------------------------------------
``` 
and 

```
------------------------------------------------------------------------
SUCCESS - Total Time: 11.232510756s
------------------------------------------------------------------------
```

Let follow the guideline specified in issue

Format as `XXm YYs` –  `If XX = 00` only print `YYs`
ex,
1. `01m 45s` 
2. `04s`
3. `21s`

- Also, round the seconds ex

`11.232510756s` to `11s`
`9.562334` to `10s`

`Round down for - <0.5 & round uo for >0.5`

File: cmd/exec/exec.go [hunk 1]
```diff
@@ -21,30 +25,29 @@ func ExecutePlan(commands ...string) {
...
-
+	utils.PrintResultsBanner(true, executeStart)
 }
 
-func executeAndStopIfFailed(command *models.CommandContext) {
+func executeAndStopIfFailed(command *models.CommandContext, executeStart time.Time) {
 	command.PrintBanner()
 	err := command.CommandSession.Run()
 	if err != nil {
 		exitCode := (err.Error())[12:]
```
Comment (gopinath-langote) [inline cmd/exec/exec.go]: The only thing missing is removing the `dashed banner` from the failure message.
Screenshot 2019-10-01 at 17 43 13

Comment (gopinath-langote) [reply cmd/exec/exec.go]: Check issue description https://github.com/gopinath-langote/1build/issues/155 for more details.

Comment (gopinath-langote) [reply cmd/exec/exec.go]: With message `Execution failed: phase 'a' - exit code: 127`

**Comment by gopinath-langote** [comment]
@budevg Thanks and congratulations on the first PR for `#Hacktoberfest`

**Pull request merged**